<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primingDialog</class>
 <widget class="QDialog" name="primingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Priming Calculator</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="SmartLabel" name="label_beerVol">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Collected Beer Volume</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_beerVol</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SmartLineEdit" name="lineEdit_beerVol">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Amount of beer to prime</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="SmartLabel" name="label_temp">
          <property name="text">
           <string>Beer Temperature</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_temp</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="SmartLineEdit" name="lineEdit_temp">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Temp of the beer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="SmartLabel" name="label_vols">
          <property name="text">
           <string>Desired Volumes</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_vols</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="SmartLineEdit" name="lineEdit_vols">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>How many volumes of CO2 you want (1 L CO2 @ STP per L beer)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_glucMono">
        <property name="text">
         <string>Glucose Monohydrate (corn sugar)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_gluc">
        <property name="text">
         <string>Anhydrous Glucose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_sucrose">
        <property name="text">
         <string>Sucrose (table sugar)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_dme">
        <property name="text">
         <string>Dry Malt Extract</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="primingdialog">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="SmartLabel" name="label_output">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Prime with</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_output</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SmartLineEdit" name="lineEdit_output">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>How much priming ingredient to use</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_calculate">
        <property name="toolTip">
         <string>Calculate</string>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
    <slot>popContextMenu(QPoint)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <signal>textModified()</signal>
    <slot>lineChanged()</slot>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_beerVol</tabstop>
  <tabstop>lineEdit_temp</tabstop>
  <tabstop>lineEdit_vols</tabstop>
  <tabstop>radioButton_glucMono</tabstop>
  <tabstop>radioButton_gluc</tabstop>
  <tabstop>radioButton_sucrose</tabstop>
  <tabstop>radioButton_dme</tabstop>
  <tabstop>lineEdit_output</tabstop>
  <tabstop>pushButton_calculate</tabstop>
 </tabstops>
 <resources/>
</ui>
